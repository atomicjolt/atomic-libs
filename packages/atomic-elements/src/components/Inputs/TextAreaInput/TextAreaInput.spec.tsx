import { test, expect } from "vitest";
import { render } from "@testing-library/react";
import { TextAreaInput } from ".";

test("matches snapshot", () => {
  const result = render(
    <TextAreaInput value="" onChange={() => {}} label="label" />
  );
  expect(result.asFragment()).toMatchSnapshot();
});
