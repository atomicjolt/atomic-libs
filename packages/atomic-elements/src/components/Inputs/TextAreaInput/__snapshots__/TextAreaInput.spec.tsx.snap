// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-gsFSjX pXqer aje-input__text sc-dLNtp hUkCVU aje-input__textarea is-medium"
    data-resize="both"
  >
    <label
      class="sc-aYaIB dImjZF"
      for="react-aria-:r0:"
      id="react-aria-:r1:"
    >
      label
    </label>
    <textarea
      class="sc-iGgVNO kYbOWo"
    />
  </div>
</DocumentFragment>
`;
